packages/maps/rollup.config.ts

```diff
@@ -0,0 +1,23 @@
+import commonjs from 'rollup-plugin-commonjs';
+import resolve from 'rollup-plugin-node-resolve';
+import sourceMaps from 'rollup-plugin-sourcemaps';
+import typescript from 'rollup-plugin-typescript2';
+
+// tslint:disable-next-line:no-var-requires
+const { modules, umd } = require('../../rollup-config-generator');
```

Comment:
Why can't we import these files with ESM syntax?

Reply:
Good question, I wish I knew.

This file is ran by rollup and I have _no idea_ with which tsconfig, so something went wrong when we wrote it in TS. We can try again and investigate.

Reply:
I'm gonna merge this without changing it now (to get back in master to a good compilation, thx to https://github.com/CartoDB/toolkit/pull/4). We can test (and fix) after that, in another PR